server/drivers/Makefile.am

```diff
@@ -24,7 +24,7 @@ AM_LDFLAGS = @LDSHARED@
 lcdexecbindir = $(pkglibdir)
 lcdexecbin_PROGRAMS = @DRIVERS@
 EXTRA_PROGRAMS = bayrad CFontz CFontzPacket curses CwLnx debug ea65 EyeboxOne futaba g15 glcd glcdlib glk hd44780 i2500vfd icp_a106 imon imonlcd IOWarrior irman irtrans joy lb216 lcdm001 lcterm linux_input lirc lis MD8800 mdm166a ms6931 mtc_s16209x MtxOrb mx5000 NoritakeVFD Olimex_MOD_LCD1x9 picolcd pyramid rawserial sdeclcd sed1330 sed1520 serialPOS serialVFD shuttleVFD sli stv5730 SureElec svga t6963 text tyan ula200 vlsys_m428 xosd yard2LCD
-noinst_LIBRARIES = libLCD.a libbignum.a
+noinst_LIBRARIES = libLCD.a libbignum.a libhidraw.a
```

Comment:
Since this will be statically linked, why bother having a library in the first place? In principle I don't care either way, in practice I'm concerned that this will cause build errors on non-linux OSes.

Reply:
The hidraw code was deliberately writtent to be generic, so putting it in a lib seems like the right thing to do to me.

Reply:
Sorry, I was being unclear: Putting this into its own file of course is the right thing to do. But it need not be a lib, it could be just an extra source file like we do it with i2c.c

I'm not opposed to doing it as lib, but then you need to make sure that it doesn't break builds on non-linux OSes, where presumably the hidraw headers are not present.

Reply:
A good point (non Linux platforms not having the linux/hidraw.h header. I will do as you suggest for the next version, dropping the libhidraw.a and instead add hidraw_lib.c to g15_SOURCES.